presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/H2TestUtil.java

```diff
@@ -186,6 +188,15 @@ private static void setup(DistributedQueryRunner queryRunner, H2ResourceGroupsDa
 
     public static List<ResourceGroupSelector> getSelectors(DistributedQueryRunner queryRunner)
     {
-        return queryRunner.getCoordinator().getResourceGroupManager().get().getConfigurationManager().getSelectors();
+        try {
+            return queryRunner.getCoordinator().getResourceGroupManager().get().getConfigurationManager().getSelectors();
+        }
+        catch (PrestoException e) {
+            if (e.getErrorCode() == CONFIGURATION_INVALID.toErrorCode()) {
```

Comment:
Why do you handle `CONFIGURATION_INVALID` here?

Reply:
Because tests rely on getSelectors returning an empty list while initializing (rather than throwing)